clients/fossid-webapp/src/main/kotlin/model/Scan.kt

```diff
@@ -20,6 +20,7 @@
 package org.ossreviewtoolkit.clients.fossid.model
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
```

Comment:
Commit message nits:
- If "String" in the title refers to the Java class, put it also in backticks (`String`).
- In 'either null, either "0"' drop the second "either".
- "This commit fixes InvalidFormatException" -> "This commit fixes an `InvalidFormatException`"